src/pages/management/configuration-api/v3.7/index.mdx

```diff
@@ -3666,6 +3667,262 @@ curl -X POST \
 
 To see the list of available webhooks v3.7, [visit the Webhooks](/management/webhooks/v3.7/) document.
 
+# Canned Responses
+
+<Section>
+
+### Create Canned Response
+
+Creates a new canned response.
+
+#### Specifics
+
+|                                  |                                                                               |
+| -------------------------------- | ----------------------------------------------------------------------------- |
+| Method URL                       | `https://api.livechatinc.com/v3.7/configuration/action/create_canned_response` |
+| Required scopes                  | `canned_responses_write`                                                          |
+| [Batch support](#batch-requests) | No                                                                            |
+
+#### Request
+
+| Parameter   | Required | Data type        | Notes                                                    |
+| ----------- | -------- | ---------------- | -------------------------------------------------------- |
+| `text`      | Yes      | `string`         | Canned response text content                         |
+| `tags`      | Yes      | `array[string]`  | Array of tags                                         |
+| `group_id`  | Yes      | `number`         | ID of the group the canned response belongs to          |
+| `is_private`| No       | `bool`           | Whether the canned response is private; default: `false`  |
+
+#### Response
+
+| Parameter | Data type | Notes                        |
+| --------- | --------- | ---------------------------- |
+| `id`      | `number`  | ID of the created canned response |
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.7/configuration/action/create_canned_response \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+    "text": "Thank you for contacting us! How can I help you today?",
+    "tags": ["greeting", "support"],
+    "group_id": 1,
+    "is_private": false
+  }'
+```
+
+</CodeSample>
+
+<CodeResponse>
+
+```json
+{
+  "id": 12345
+}
+```
+
+</CodeResponse>
+
+</Code>
+
+### Update Canned Response
+
+Updates an existing canned response. All parameters are optional - only provided parameters will be updated.
```

Comment:
i wonder if we should write that _all_ parameters are optional, considering that the `id` is not